include/qpdf/Buffer.hh

```diff
@@ -56,6 +56,13 @@ class Buffer
     QPDF_DLL
     unsigned char* getBuffer();
 
+    // Create a new copy of the Buffer. The new Buffer owns an independent copy of the data.
+    QPDF_DLL
+    Buffer copy() const;
+
+    // Only used during CI testing.
```

Comment:
Please add an ABI comment with a reminder to remove this when we remove the deprecated operator=.

Reply:
Done